lib/CMakeLists.txt

```diff
@@ -62,7 +61,6 @@ LIST(REMOVE_ITEM QUDA_OBJS ${QUDA_CU_OBJS})
 
 if(BUILD_FORTRAN_INTERFACE)
   LIST(APPEND QUDA_OBJS quda_fortran.F90)
-  set_source_files_properties(quda_fortran.F90 PROPERTIES OBJECT_OUTPUTS ${CMAKE_CURRENT_BINARY_DIR}/quda_fortran.mod)
```

Comment:
Good catch. I guess @weinbe2 this was an accident in 7055588d7c920fbdd4bbb752ff032c94d29d1dae?

I'll restore it.